public/index.html

```diff
@@ -8,6 +8,7 @@
 
     <!-- Lato regular, regular italic -->
     <link href="https://fonts.googleapis.com/css2?family=Lato:ital@0;1&display=swap" rel="stylesheet"> 
+    <script src="https://kit.fontawesome.com/e533f8d3fb.js" crossorigin="anonymous"></script>
```

Comment:
Is this meant to resolve #46 ?

Reply:
I was thinking of resolving #46 in another PR to keep this one on scope. We might have a dangling font awesome lying there but I will make the next PR immediately after this one goes in.

Reply:
Sounds good! 🎉